### DISCUSSION TITLE
Sku in OrchardCore.Cms.Web

### DISCUSSION THREAD

**Discussion by havana7** [OrchardCMS/OrchardCore.Commerce#277] [Q&A]
HI.
I added the OrchardCore.Commerce module to the OrchardCore.Cms.Web project to add the ecommerce functions.
I activated the Commerce features and created a new content type and added ProductPart but the Sku field is not showing in edit product.
Can you tell me why?

Thank you.

**Comment by sarahelsaig** [comment]
I can't reproduce it. Can you give more details and a step-by-step repro?

I just created a new site, enabled these features:
![image](https://user-images.githubusercontent.com/4007293/229489886-f80c1d4e-3d48-4afa-be63-7e0f0b8fe820.png)

Then created a new content type called Product and added the Product content part. (of course the Price part should be added as well to make it actually functional)
![image](https://user-images.githubusercontent.com/4007293/229490111-5f4281ae-33b2-4998-99b0-44d4bd0d37e0.png)

So when I create a new Product content item, the SKU field is there in the editor:
![image](https://user-images.githubusercontent.com/4007293/229490336-e5a76a3a-f75f-4fdc-bf83-b7662e46cb41.png)

**Comment by havana7** [comment]
I created a new database and a new tenant on OrchardCore.Cms.Web and this is the result

![image](https://user-images.githubusercontent.com/394454/229507792-82f04f65-1bc0-4239-8318-2109008b3201.png)
![image](https://user-images.githubusercontent.com/394454/229517332-c46bf172-c2c2-4264-9f07-8fc59b23afcd.png)
![image](https://user-images.githubusercontent.com/394454/229517564-0ffeb77a-f7ff-48ee-b861-0875de141847.png)
![image](https://user-images.githubusercontent.com/394454/229517733-37857b15-2395-4978-9d67-769652bd7d8d.png)
